Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "financial_snapshot_contract"
-version = "0.0.0"
+version = "0.1.0"
```

Comment:
@kenta-elys we should keep this value to 0.0.0 this is using the best practices rust/cargo community’s using to speed the release process, so no need to increment this number anymore

Reply:
@kenta-elys never mind I did the change